test/integration/api.bats

```diff
@@ -13,24 +13,39 @@ function teardown() {
 }
 
 @test "docker build" {
-      start_docker 3
```

Comment:
Watch out for tabs / spaces - I believe we are using tabs in `*.bats` files and your editor replaced them by spaces.